environments/prod/prod.tfvars

```diff
@@ -506,5 +506,12 @@ frontends = [
     custom_domain    = "www.apply-for-probate.service.gov.uk"
     backend_domain   = ["firewall-prod-int-palo-prod.uksouth.cloudapp.azure.com"]
     certificate_name = "apply-for-probate-service-gov-uk"
+  },
+  {
+    name             = "ia-aip"
+    mode             = "Detection"
+    custom_domain    = "immigration-appeal.platform.hmcts.net"
```

Comment:
it seems that in your public DNS PR you've set it up with www, but here you haven't, which is correct?

https://github.com/hmcts/azure-public-dns/blob/master/environments/prod/platform-hmcts-net.yml#L605-L608
https://github.com/hmcts/azure-public-dns/blob/master/environments/prod/platform-hmcts-net.yml#L618-L620

Reply:
Hmm - should be without the www.  Shall I change the DNS one?

Reply:
will is cause a problem since the DNS one is merged

Reply:
changed the dns config